lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -39,12 +39,7 @@
                     user: user,
                     enrollments: this.enrollments,
                     formatDate: function(date) {
-                        if (!date) {
-                            return 'N/A';
-                        }
-                        else {
-                            return moment(date).format('MM/DD/YYYY (H:MM UTC)');
-                        }
+                        return date ? moment.utc(date).format('lll z') : 'N/A';
```

Comment:
@mushtaqak For maintaining the consistency of date format.
See this:
https://github.com/edx/ecommerce/blob/0f970756bde63b2a0ff7e129adb071c676893c7a/ecommerce/static/js/views/course_detail_view.js#L40-L40